specification/schemas/ShipmentStatus.json

```diff
@@ -32,6 +30,11 @@
               "type": "number",
               "example": 1504801719,
               "description": "Unix timestamp when the carrier registered this status."
+            },
+            "created_at": {
+              "type": "number",
+              "example": 1504801719,
+              "description": "Unix timestamp when the status was created."
```

Comment:
For created_at and updated_at we use this everywhere:
```
"created_at": {
  "$ref": "#/components/schemas/Timestamp"
}
```

Reply:
We should probably use that for `carrier_timestamp` as well then. Agreed?